src/libraries/Microsoft.PowerFx.Core/Localization/ErrorResourceKey.cs

```diff
@@ -15,11 +15,22 @@ public struct ErrorResourceKey
     {
         public string Key { get; }
 
+        internal IResourceStringManager ResourceManager { get; }
```

Comment:
>internal [](http://example.com/codeflow?start=8&length=8)

This is the key change here. 
Since we can now pull resources from multiple dlls, we need some scoping to find the right dll.